src/utils/validateIdentifierInputs.test.ts

```diff
@@ -1,7 +1,7 @@
 import { validateIdentifierInputs } from "./validateIdentifierInputs";
 
 describe("validateIdentifierInputs", () => {
-  it("should return true if all identifiers are valid", () => {
+  it("should return validated input if all identifiers are valid", () => {
```

Comment:
do revert this to true 
```suggestion
  it("should return true if all identifiers are valid", () => {
```